events_task/priv/repo/migrations/20191212174808_create_events.exs

```diff
@@ -0,0 +1,15 @@
+defmodule MyApp.Repo.Migrations.CreateEvents do
+  use Ecto.Migration
+
+  def change do
+    create table(:events) do
+      add(:description, :string)
+      add(:timestamp, :naive_datetime)
+      add(:author_id, references(:users, on_delete: :nothing))
```

Comment:
`on_delete: :nothing` is always a bad choice because it doesn't help you keep DB data consistent.
You should always choose from one of other options:
* `:delete_all` if you want all user events to be deleted when a user is deleted
* `:nilify_all` if you events can exist without an author
* `:restrict` if you don't want to allow deleting a user when it has some events